src/model/AirLoopHVACUnitaryHeatPumpAirToAirMultiSpeed.cpp

```diff
@@ -1229,5 +1228,29 @@ namespace model {
     return getImpl<detail::AirLoopHVACUnitaryHeatPumpAirToAirMultiSpeed_Impl>()->autosizedSpeed4SupplyAirFlowRateDuringCoolingOperation();
   }
 
+  // DEPRECATED
+  double AirLoopHVACUnitaryHeatPumpAirToAirMultiSpeed::minimumOutdoorDryBulbTemperatureforCompressorOperation() const {
+    // Former v25.1.0 IDD note said this "Needs to match the corresponding minimum outdoor temperature defined in the DX heating coil object"
+    DEPRECATED_AT_MSG(3, 11, 0,
+                      "This field was previously unused and removed in EnergyPlus 25.2.0. Set the Minimum OAT for Compressor Operation on the child "
+                      "DX MultiSpeed Coils if any.");
+    if (auto hc_ = heatingCoil().optionalCast<CoilHeatingDXMultiSpeed>()) {
+      return hc_->minimumOutdoorDryBulbTemperatureforCompressorOperation();
+    }
+    return -1000.0;
+  }
+
+  bool AirLoopHVACUnitaryHeatPumpAirToAirMultiSpeed::setMinimumOutdoorDryBulbTemperatureforCompressorOperation(
+    double minimumOutdoorDryBulbTemperatureforCompressorOperation) {
+    DEPRECATED_AT_MSG(3, 11, 0,
+                      "This field was previously unused and removed in EnergyPlus 25.2.0. Set the Minimum OAT for Compressor Operation on the child "
+                      "DX MultiSpeed Coils if any.");
+    if (auto hc_ = heatingCoil().optionalCast<CoilHeatingDXMultiSpeed>()) {
+      return hc_->setMinimumOutdoorDryBulbTemperatureforCompressorOperation(minimumOutdoorDryBulbTemperatureforCompressorOperation);
+    }
+
+    return false;
+  }
```

Comment:
@joseph-robertson My proposed implementation here.

I picked -1000.0 because that's what E+ GetMinOATCompressor does.

https://github.com/NREL/EnergyPlus/blob/0befe779fe9721745ba31a65f877cc05905ac6b3/src/EnergyPlus/DXCoils.cc#L15514-L15534